### PR TITLE
BrenKt-SK858-TRANS-WEB-Email-alloffers-change_password

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/emails/email_change_password/guide.de-de.md [hunk 1]
```diff
@@ -1,58 +1,91 @@
...
+
+> [!warning]
+> Aus Sicherheitsgründen empfehlen wir Ihnen, nicht zweimal das gleiche Passwort zu verwenden, eines auszuwählen, das nichts mit Ihren persönlichen Angaben zu tun hat (vermeiden Sie beispielsweise die Angaben zu Ihrem Namen, Vornamen und Geburtsdatum) und es regelmäßig zu verlängern.
+
+> [!tabs]
+> **MXplan E-Mails (historische Version)**
+>>
+>> Wenn Sie den Typ des MXplan Angebots nicht kennen, lesen Sie bitte unseren Abschnitt ><br>
+>> Klicken Sie auf `E-Mails`{.action} und wählen Sie den Namen des betreffenden MX Plan Dienstes aus. Gehen Sie auf den Tab `E-Mails`{.action}. Im angezeigten Fenster werden die bestehenden E-Mail-Accounts angezeigt. <br>
+>> Klicken Sie auf <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i>den Button und `dann auf Passwort {.action}` ändern.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.de-de.md]: {.action} format is broken

File: pages/web/emails/email_change_password/guide.de-de.md [hunk 2]
```diff
@@ -1,58 +1,91 @@
...
+>>![{.thumbnail}](images/email-password-mxplan-legacy01.png) E-Mail<br>
+>>
+> **MXplan E-Mails (neue Version)**
+>>
+>> Wenn Sie den Typ des MXplan Angebots nicht kennen, lesen Sie bitte unseren Abschnitt ><br>
+>> Klicken Sie auf `E-Mails`{.action} und wählen Sie den Namen des betreffenden MX Plan Dienstes aus. Gehen Sie auf den Tab `E-Mails`{.action}. Im angezeigten Fenster werden die bestehenden E-Mail-Accounts angezeigt. <br>
+>> Klicken Sie auf den Button <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i>und dann auf `Ändern`{.action}.<br><br>
+>>![{.thumbnail}](images/email-password-mxplan-new01.png) E-Mail<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.de-de.md]: thumbnail part should not be at the beginning

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 3]
```diff
@@ -1,62 +1,95 @@
...
+- From Webmail
+- Via the OVHcloud Control Panel
+
+**Find out how to change the password for an OVHcloud email address.**
 
 ## Requirements
 
-- an MX Plan solution, available as part of our [web hosting plans](https://www.ovhcloud.com/en-gb/web-hosting/){.external}, the [free Start 10M hosting plan](https://www.ovhcloud.com/en-gb/domains/free-web-hosting/){.external}, or ordered separately as a standalone solution
-- access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external} in the `Web Cloud`{.action} section, or access to the email address via Outlook Web Messenger (OWA) [webmail](https://www.ovh.co.uk/mail/){.external}
+- You must have a pre-configured OVHcloud email solution (**MX Plan**, included in our web[ hosting ](https://www.ovhcloud.com/en-gb/web-hosting/)plans, included in a free[ Start10M ](https://www.ovhcloud.com/en-gb/domains/free-web-hosting/) hosting package, or ordered separately as a standalone solution, such as [**Hosted Exchange**](https://www.ovhcloud.com/en-gb/emails/hosted-exchange/) or [**Email Pro**](https://www.ovhcloud.com/en-gb/emails/email-pro/)).
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
- You must have a pre-configured OVHcloud email solution (**MX Plan**, included in our [web hosting plans](https://www.ovhcloud.com/en-gb/web-hosting/), included in a free [Start10M](https://www.ovhcloud.com/en-gb/domains/free-web-hosting/) hosting package, or ordered separately as a standalone solution, such as [**Hosted Exchange**](https://www.ovhcloud.com/en-gb/emails/hosted-exchange/) or [**Email Pro**](https://www.ovhcloud.com/en-gb/emails/email-pro/)).
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 4]
```diff
@@ -1,62 +1,95 @@
...
-- [Change the password via the OVHcloud Control Panel](./#change-the-password-via-the-ovhcloud-control-panel). You will not need your current password for this method.
+### Change the password via the Control Panel <a name="controlpanel"></a>
+
+> [!warning]
+> For security reasons, we recommend not using the same password twice, and choosing one that does not contain any personal information (e.g. your surname, first name and date of birth). We also recommend renewing your password regularly.
+
+> [!tabs]
+> **MXplan emails (historical version)**
+>>
+>> If you do not know the type of MXplan solution you have, please read our ><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]: Part of translation is missing here

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 5]
```diff
@@ -1,62 +1,95 @@
...
+>> Click `Emails`{.action}, then choose the name of the MX Plan service concerned. Go to the {.action} `Emails` tab. The window that appears will display the existing email accounts. <br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
>> Click `Emails`{.action}, then choose the name of the MX Plan service concerned. Go to the `Emails`{.action} tab. The window that appears will display the existing email accounts. <br>
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 6]
```diff
@@ -1,62 +1,95 @@
...
+>> Click the button, <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i>then click `Change password`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
>> Click the <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i> button, then click `Change password`{.action}.<br><br>
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 7]
```diff
@@ -1,62 +1,95 @@
...
+>>![email](images/email-password-mxplan-legacy01.png){.thumbnail}<br>
+>>
+> **MXplan emails (new version)**
+>>
+>> If you do not know the type of MXplan solution you have, please read our ><br>
+>> Click `Emails`{.action}, then choose the name of the MX Plan service concerned. Go to the {.action} `Emails` tab. The window that appears will display the existing email accounts. <br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
>> Click `Emails`{.action}, then choose the name of the MX Plan service concerned. Go to the `Emails`{.action} tab. The window that appears will display the existing email accounts. <br>
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 8]
```diff
@@ -1,62 +1,95 @@
...
+>> Click the button and <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i>then click `Edit`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
>> Click the <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i> button then click `Edit`{.action}.<br><br>
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 9]
```diff
@@ -1,62 +1,95 @@
...
+>>![email](images/email-password-mxplan-new01.png){.thumbnail}<br>
+>>
+> **Email Pro**
+>>
+>> Click `Email Pro`{.action}, then choose the name of the platform concerned. Go to the `Email`{.action} accounts tab. The window that appears will display the existing email accounts.<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
>> Click `Email Pro`{.action}, then choose the name of the platform concerned. Go to the `Email accounts`{.action}  tab. The window that appears will display the existing email accounts.<br>
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 10]
```diff
@@ -1,62 +1,95 @@
...
+>> Click the button and <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i>then click `Edit`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
>> Click the <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i> button then click `Edit`{.action}.<br><br>
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 11]
```diff
@@ -1,62 +1,95 @@
...
+>>![email](images/email-password-emailpro01.png){.thumbnail}<br>
+>>
+> **Exchange**
+>>
+>> Click `Microsoft`{.action} / `Exchange`{.action}, then choose the name of the platform concerned. Go to the `Email`{.action} accounts tab. The window that appears will display the existing email accounts.<br>
+>> Click the button and <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i>then click `Edit`{.action}.<br><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
>> Click the <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i> button then click `Edit`{.action}.<br><br>
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 12]
```diff
@@ -1,62 +1,95 @@
...
+
+Changing your password via webmail is available for OVHcloud email offers using **OWA** (**O**utlook **W**eb **A**pplication):
+
+- MXplan emails (new version)
+- Email Pro
+- Exchange
+
+> [!warning]
+>
+> The MXplan **emails solution (historical version)** uses Roundcube webmail. You can only change your password [via the OVHcloud Control Panel](#controlpanel).
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]: Please check where bold should apply here

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 13]
```diff
@@ -1,62 +1,95 @@
...
-#### Change the password via OWA webmail.
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/msmUN7cLSNI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-Firstly, go to the [webmail login](https://www.ovh.co.uk/mail/){.external} page. Enter your full email address and password, then click the `Login`{.action} button. 
+Go to the [Webmail](https://www.ovhcloud.com/en-gb/mail/) page. Enter your full email address and password, then click the `Login`{.action} button. 
 
 ![email](images/mxplan-password-new-step2.png){.thumbnail}
 
-Your inbox will appear. Click the cogwheel icon at the top, then click `Options`{.action} .
+Click the button <i class="icons-gear-concept icons-masterbrand-blue"></i>at the top, then click `Options`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
Click the <i class="icons-gear-concept icons-masterbrand-blue"></i> button at the top, then click `Options`{.action}.
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 14]
```diff
@@ -1,62 +1,95 @@
...
 
 ![email](images/mxplan-password-new-step3.png){.thumbnail}
 
-On the left-hand side of the new page that opens, expand the "General" tab in the tree-view, then click `My account`{.action} . Next, click `Change your password`{.action}.
+In the left-hand side of the new page that opens, expand the "General" tab in the tree-view, then click `My account`{.action} . Next, click `Change password`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]:
```suggestion
On the left-hand side of the new page that opens, expand the "General" tab in the tree-view, then click `My account`{.action}. Next, click `Change password`{.action}.
```

File: pages/web/emails/email_change_password/guide.en-gb.md [hunk 15]
```diff
@@ -69,48 +102,22 @@ In the new window that opens, enter your current password first. Then enter your
...
-![email](images/mxplan-password-legacy-step2.png){.thumbnail}
-
-In the window that opens, enter a new password and confirm it. For security reasons, we recommend not using the same password twice, choosing one that does not contain any personal information (e.g. your surname, first name, and date of birth), and we also recommend renewing it regularly.
-
-Once you have done this, click `Confirm`{.action}. The password change will take effect after a few minutes.
-
-![email](images/mxplan-password-legacy-step3.png){.thumbnail}
+|MX Plan legacy version|MX Plan new version|
+|---|---|
+|![email](images/mxplan-starter-legacy-step1.png){.thumbnail}<br> The solution appears in the “**Subscription**” box on the right, the legacy solution *does not* have a server reference.|![email](images/mxplan-starter-new-step1.png){.thumbnail}<br>The new solution has a \`Server **model**\` in the \`**Summary**\` box on the left.|
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.en-gb.md]: Formatting here needs to be checked

File: pages/web/emails/email_change_password/guide.es-es.md [hunk 16]
```diff
@@ -1,66 +1,99 @@
 ---
-title: 'Cambiar la contraseña de una dirección de correo MX Plan'
+title: 'Cambiar la contraseña de una dirección de correo'
 slug: cambiar-contrasena-direccion-correo
-excerpt: 'Cómo cambiar la contraseña de una dirección de correo electrónico de la solución MX Plan'
+excerpt: 'Cómo cambiar la contraseña de una dirección de correo de OVHcloud'
 section: 'Primeros pasos'
 order: 03
 ---
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.es-es.md:8]: Please add Systran translation warning to all translations.

File: pages/web/emails/email_change_password/guide.es-es.md [hunk 17]
```diff
@@ -1,66 +1,99 @@
...
-- [Cambiar la contraseña desde el área de cliente de OVHcloud](./#cambiar-la-contrasena-desde-el-area-de-cliente-de-ovh): En este caso, no necesitará la contraseña actual.
+### Cambiar la contraseña desde el área de cliente <a name="controlpanel"></a>
+
+> [!warning]
+> Por motivos de seguridad, le recomendamos que no utilice dos veces la misma contraseña, que elija una que no guarde ninguna relación con sus datos personales (evite mencionar su nombre, apellidos y fecha de nacimiento, por ejemplo) y que la cambie periódicamente.
+
+> [!tabs]
+> **MXplan (versión histórica)**
+>>
+>> Si no conoce el tipo de producto MXplan que tiene, consulte el apartado ><br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.es-es.md]: Missing part here, check it for all translations.

File: pages/web/emails/email_change_password/guide.es-es.md [hunk 18]
```diff
@@ -1,66 +1,99 @@
...
+> [!warning]
+>
+> La solución **MXplan (versión histórica)** utiliza el webmail Roundcube. La modificación de la contraseña solo se realiza [desde el área de cliente](#controlpanel).
+>
 
-#### Cambiar la contraseña desde el webmail Outlook en la Web
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/xnq6wvANUFs" title="Vídeo de YouTube" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-Vaya a la [página de conexión al webmail](https://www.ovh.es/mail/){.external} e introduzca su dirección de correo electrónico completa y la contraseña actual. Haga clic en el botón `Conexión`{.action}. 
+Acceda a la página "[Webmail](https://www.ovhcloud.com/es/mail/)". e introduzca su dirección de correo electrónico completa y la contraseña actual. Haga clic en el botón `Conexión`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.es-es.md]:
```suggestion
Acceda a la página [Webmail](https://www.ovhcloud.com/es/mail/) e introduzca su dirección de correo electrónico completa y la contraseña actual. Haga clic en el botón `Conexión`{.action}. 
```

File: pages/web/emails/email_change_password/guide.it-it.md [hunk 19]
```diff
@@ -1,60 +1,95 @@
...
-Per modificare la password di un account email è possibile procedere in due modi diversi. Prosegui nella lettura di questa guida in base alla modalità che intendi utilizzare.
-
-- [Modificare la password dalla Webmail <i>Outlook on the web</i>](./#modificare-la-password-dalla-webmail-owa): in questo caso è necessario disporre della password corrente. 
-- [Modificare la password dallo Spazio Cliente OVHcloud](./#modificare-la-password-dallo-spazio-cliente-ovh): in questo caso non è necessario disporre della password corrente.
+### Modifica la password dallo Spazio Cliente OVH <a name="controlpanel"></a>
+
+> [!warning]
+> Per motivi di sicurezza, ti consigliamo di non utilizzare due volte la stessa password, sceglierne una che non ha alcun rapporto con le tue informazioni personali (ad esempio, eviti le indicazioni del tuo cognome, nome e data di nascita) e rinnovarla regolarmente.
+
+> \[!tabs]
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.it-it.md]:
```suggestion
> [!tabs]
```

File: pages/web/emails/email_change_password/guide.it-it.md [hunk 20]
```diff
@@ -1,60 +1,95 @@
...
+> [!warning]
+>
+> L'offerta **Email MXplan (versione storica)** utilizza Roundcube, la modifica della password può essere effettuata solo [dallo Spazio Cliente](#controlpanel).
+>
 
-#### Modificare la password dalla Webmail <i>Outlook on the web</i>
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/msmUN7cLSNI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-Accedi alla [pagina di connessione](https://www.ovh.it/mail/){.external}, inserisci le tue credenziali e clicca sul pulsante `Connessione`{.action}. 
+Accedi alla pagina "[Webmail](https://www.ovhcloud.com/it/mail/)". inserisci le tue credenziali e clicca sul pulsante `Connessione`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.it-it.md]:
```suggestion
Accedi alla pagina [Webmail](https://www.ovhcloud.com/it/mail/), inserisci le tue credenziali e clicca sul pulsante `Connessione`{.action}. 
```

File: pages/web/emails/email_change_password/guide.pl-pl.md [hunk 21]
```diff
@@ -1,115 +1,123 @@
...
+
+**Dowiedz się, jak zmienić hasło do konta e-mail OVHcloud.**
 
 ## Wymagania początkowe
 
-- Wykupienie usługi MX Plan Jest ona dostępna w ramach oferty: [Hosting](https://www.ovhcloud.com/pl/web-hosting/){.external}, [Hosting Start 10M](https://www.ovhcloud.com/pl/domains/free-web-hosting/){.external} lub oferty MX Plan zamówionej oddzielnie.
-- W zależności od metody, z której korzystasz: zalogowanie do [Panelu klienta](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl){.external} > sekcja `Web Cloud`{.action} lub zalogowanie do konta e-mail za pośrednictwem interfejsu [Webmail](https://www.ovh.pl/mail/){.external} Outlook Web App (OWA).
+- Posiadanie wcześniej skonfigurowanego rozwiązania poczty elektronicznej OVHcloud (**MX Plan**, zaproponowanego w naszej [ofercie hostingu](https://www.ovhcloud.com/fr/web-hosting/), zawartego w [hostingu Start10M bezpłatnie](https://www.ovhcloud.com/fr/domains/free-web-hosting/) lub zamówionym oddzielnie jako rozwiązanie autonomiczne, takie jak [**Hosted Exchange**](https://www.ovhcloud.com/fr/emails/hosted-exchange/) lub [**Email Pro**](https://www.ovhcloud.com/fr/emails/email-pro/))
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.pl-pl.md]: FR links

File: pages/web/emails/email_change_password/guide.pl-pl.md [hunk 22]
```diff
@@ -1,115 +1,123 @@
...
+- W zależności od metody, której używasz: zalogować się do [Panelu klienta OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.pl/&ovhSubsidiary=pl) w sekcji `Web Cloud`{.action} lub logować się do konta e-mail za pośrednictwem [interfejsu webmail](https://www.ovhcloud.com/pl/mail/).
 
 > [!primary]
 >
-> Po zmianie hasła do Twojego konta e-mail wprowadź je na wszystkich urządzeniach, na których skonfigurowałeś konto.
+> Po zmianie hasła do Twojego konta e-mail wprowadź tę zmianę na wszystkich urządzeniach, na których skonfigurowałeś konto. Zapoznaj się z naszymi przewodnikami dotyczącymi konfiguracji Twojego programu pocztowego na stronie głównej Twojej usługi e-mail:
+>
+> - [MXplan](https://docs.ovh.com/fr/emails/)
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.pl-pl.md]: FR links

File: pages/web/emails/email_change_password/guide.pt-pt.md [hunk 23]
```diff
@@ -1,58 +1,91 @@
...
+> [!warning]
+> Por razões de segurança, recomendamos que não utilize duas vezes a mesma palavra-passe, selecione uma palavra-passe que não tenha qualquer relação com as suas informações pessoais (evite as menções do seu apelido, nome próprio e data de nascimento, por exemplo) e que a renove regularmente.
+
+> [!tabs]
+> **E-mails MXplan (versão histórica)**
+>>
+>> Se não conhece o tipo de oferta MXplan que possui, consulte o nosso parágrafo ><br>
+>> Clique em `E-mails`{.action} e escolha o nome do serviço MX Plan em causa. Aceda ao separador `E-mails`{.action}. Na nova janela, podem ver-se as contas de e-mail existentes. <br>
+>> Clique no botão <i class="icons-elipsis icons-border-rounded icons-masterbrand-blue"></i>e, a seguir, em `Alterar a palavra-passe`{.action}.<br><br>
+>>![e-mail](images/email-password-mxplan-legacy01.png) {.thumbnail}<br>
```
Comment (OVH-Team-Guides) [inline pages/web/emails/email_change_password/guide.pt-pt.md]:
```suggestion
>>![e-mail](images/email-password-mxplan-legacy01.png){.thumbnail}<br>
```

Comment (OVH-Team-Guides) [reply pages/web/emails/email_change_password/guide.pt-pt.md]: Search for extra spaces before {.thumbnail} on all versions

**Review by OVH-Team-Guides — DISMISSED** [review]

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**